docs/Displaying a User Interface/Main Screen Templates/index.md

```diff
@@ -134,6 +134,255 @@ if (success) {
+@interface HomeSDLScreen : NSObject<CustomSDLScreen>
+
+@end
+
+NS_ASSUME_NONNULL_END
+
+// HomeSDLScreen.m
+#import "HomeSDLScreen.h"
+
+#import "CustomSDLScreen.h"
+#import "HomeDataModel.h"
+#import "SmartDeviceLink.h"
+
+@interface HomeSDLScreen()
+
+@property (strong, nonatomic) SDLManager *sdlManager;
+@property (assign, nonatomic) HomeDataModel *homeDataModel;
+
+@end
+
+@implementation HomeSDLScreen
+
+- (instancetype)initWithManager:(SDLManager *)sdlManager {
+    _sdlManager = sdlManager;
+    _homeDataModel = HomeDataModel();
+}
+
+- (void)showScreen {
+    [self.sdlManager.screenManager beginUpdates];
+    [self.sdlManager.screenManager changeLayout:[[SDLTemplateConfiguration alloc] initWithTemplate:SDLPredefinedLayoutGraphicWithText] withCompletionHandler:nil];
+    self.sdlManager.screenManager.textField1 = self.homeDataModel.text1;
+    self.sdlManager.screenManager.textField2 = self.homeDataModel.text2;
+    self.sdlManager.screenManager.textField3 = self.homeDataModel.text3;
+    self.sdlManager.screenManager.textField4 = self.homeDataModel.text4;
+    self.sdlManager.screenManager.softButtonObjects = @[];
+    self.sdlManager.screenManager.primaryGraphic = <#SDLArtwork#>;
+    [self.sdlManager.screenManager endUpdates];
+}
+
+@end
+
+// ButtonSDLScreen.h
+#import <Foundation/Foundation.h>
+#import "CustomSDLScreen.h"
+
+@class ButtonDataModel;
+@class SDLManager;
+@class SDLScreenManager;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ButtonSDLScreen : NSObject<CustomSDLScreen>
+
+@end
+
+NS_ASSUME_NONNULL_END
+
+// ButtonSDLScreen.m
+#import "ButtonSDLScreen.h"
+
+#import "ButtonDataModel.h"
+#import "CustomSDLScreen.h"
+#import "SmartDeviceLink.h"
+
+@interface ButtonSDLScreen()
+
+@property (strong, nonatomic) SDLManager *sdlManager;
+@property (assign, nonatomic) ButtonDataModel *buttonDataModel;
+
+@end
+
+@implementation ButtonSDLScreen
+
+- (instancetype)initWithManager:(SDLManager *)sdlManager {
+    _sdlManager = sdlManager;
+    _buttonDataModel = ButtonDataModel();
+}
+
+- (void)showScreen {
+    [self.sdlManager.screenManager beginUpdates];
+    [self.sdlManager.screenManager changeLayout:[[SDLTemplateConfiguration alloc] initWithTemplate:SDLPredefinedLayoutTilesOnly] withCompletionHandler:nil];
+    self.sdlManager.screenManager.textField1 = self.buttonDataModel.text1;
+    self.sdlManager.screenManager.textField2 = self.buttonDataModel.text2;
+    self.sdlManager.screenManager.textField3 = self.buttonDataModel.text3;
+    self.sdlManager.screenManager.textField4 = self.buttonDataModel.text4;
+    self.sdlManager.screenManager.softButtonObjects = self.buttonDataModel.buttons;
+    [self.sdlManager.screenManager endUpdates];
+}
+
+@end
+
+// DataSDLScreen.h
+#import <Foundation/Foundation.h>
+#import "CustomSDLScreen.h"
+
+@class DataModel;
+@class SDLManager;
+@class SDLScreenManager;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DataSDLScreen : NSObject<CustomSDLScreen>
+
+@end
+
+NS_ASSUME_NONNULL_END
+
+// DataSDLScreen.m
```

Comment:
What's the difference between this and the home view model? I think this could be removed.